app/cafe.py

```diff
@@ -0,0 +1,26 @@
+from datetime import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor is not vaccined.")
+
+        expiration_date = visitor["vaccine"].get("expiration_date")
+
+        if not expiration_date:
+            raise OutdatedVaccineError("Vaccine expiration date is missing.")
+
+        if expiration_date < datetime.now().date():
+            raise OutdatedVaccineError("Vaccine is expirited.")
+
+        if not visitor.get("wear masks", False):
```

Comment:
The key 'wear masks' in the visitor dictionary should be consistent with the expected key. Consider using 'wear_mask' or 'wearing_mask' to avoid confusion.